src/commands/cmd_server.cc

```diff
@@ -309,6 +309,44 @@ class CommandDBSize : public Commander {
   }
 };
 
+class CommandProfiling : public Commander {
+ public:
+  Status Parse(const std::vector<std::string> &args) override {
+    CommandParser parser(args, 1);
+    if (!parser.EatEqICase("memory")) {
+      return {Status::NotOK, "only supports MEMORY subcommand"};
+    }
+    if (parser.EatEqICase("enabled")) {
+      enabled_ = true;
+    } else if (parser.EatEqICase("disabled")) {
+      enabled_ = false;
```

Comment:
```suggestion
    if (parser.EatEqICase("enable")) {
      enabled_ = true;
    } else if (parser.EatEqICase("disable")) {
      enabled_ = false;
```

action instead of status?

Reply:
It's expected to be a status instead of an action, perhaps we could rename it to `ON|OFF` to make it clearer?

Reply:
nope. we changed the current status via `profiling memory enabled`, and this is called an "action".

Reply:
if we use "enabled" here, this is a status and we should not change it.

note that "enable" means "make it enabled".

Reply:
Makes sense. I was initially expecting this to be a set status because I think the verb should be the first word(enable memory profiling).